tests/puppeteer/campaigns/sanity/03_ordersBO/01_filterOrders.js

```diff
@@ -43,48 +44,79 @@ describe('Filter the Orders table by ID, REFERENCE, STATUS', async () => {
   it('should go to the Orders page', async function () {
     await this.pageObjects.boBasePage.goToSubMenu(
       this.pageObjects.boBasePage.ordersParentLink,
-      this.pageObjects.orderPage.ordersLink,
+      this.pageObjects.boBasePage.ordersLink,
     );
-    const pageTitle = await this.pageObjects.orderPage.getPageTitle();
-    await expect(pageTitle).to.contains(this.pageObjects.orderPage.pageTitle);
+    const pageTitle = await this.pageObjects.ordersPage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.ordersPage.pageTitle);
+  });
+  it('should reset all filters and get number of orders', async function () {
+    await this.pageObjects.ordersPage.resetFilter();
+    numberOfOrders = await parseInt(
+      await this.pageObjects.ordersPage.getTextContent(this.pageObjects.ordersPage.ordersNumberSpan),
+      10,
+    );
+    await expect(numberOfOrders).to.be.above(0);
   });
   it('should filter the Orders table by ID and check the result', async function () {
-    await this.pageObjects.orderPage.filterTableByInput(
-      this.pageObjects.orderPage.orderFilterIdInput,
+    await this.pageObjects.ordersPage.filterOrders(
+      'input',
+      'id_order',
       Orders.firstOrder.id,
-      this.pageObjects.orderPage.searchButton,
     );
-    const result = await this.pageObjects.boBasePage.checkTextValue(
-      this.pageObjects.orderPage.orderfirstLineIdTD,
+    const result = await this.pageObjects.ordersPage.checkTextValue(
+      this.pageObjects.ordersPage.orderfirstLineIdTD.replace('%ROW', '1'),
       Orders.firstOrder.id,
     );
     await expect(result).to.be.true;
-    await this.pageObjects.orderPage.waitForSelectorAndClick(this.pageObjects.orderPage.resetButton);
+  });
+  it('should reset all filters', async function () {
+    await this.pageObjects.ordersPage.resetFilter();
+    const numberOfOrdersAfterReset = await parseInt(
+      await this.pageObjects.ordersPage.getTextContent(this.pageObjects.ordersPage.ordersNumberSpan),
+      10,
+    );
+    await expect(numberOfOrdersAfterReset).to.be.equal(numberOfOrders);
   });
   it('should filter the Orders table by REFERENCE and check the result', async function () {
-    await this.pageObjects.orderPage.filterTableByInput(
-      this.pageObjects.orderPage.orderFilterReferenceInput,
+    await this.pageObjects.ordersPage.filterOrders(
+      'input',
+      'reference',
       Orders.fourthOrder.ref,
-      this.pageObjects.orderPage.searchButton,
     );
     const result = await this.pageObjects.boBasePage.checkTextValue(
-      this.pageObjects.orderPage.orderfirstLineReferenceTD,
+      this.pageObjects.ordersPage.orderfirstLineReferenceTD.replace('%ROW', '1'),
```

Comment:
Why '1'?

Reply:
> Why '1'

We are sure that we will have a result (and it will be in first row)
> Why I add `.replace('%ROW', '1')`

Selector took first row by default, I changed it in orders page, now he have to choose in which row the check should be.
> Why string and not an Int

Can be an int